app/views/foods/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<p style="color: green"><%= notice %></p>
+<%= link_to "Add Food", new_food_path, class: "add-food-link" %>
+
+  <table>
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Measurement Unit</th>
+        <th>Unit Price</th>
+        <th>Actions</th>
+      </tr>
+    </thead>
+
+    <tbody>
+      <% @foods.each do |food| %>
+        <%= render food %>
+        <tr>
+          <td><%= food.name %></td>
+          <td><%= food.measurement_unit %></td>
+          <td>$<%= food.price %></td>
+          <td><%= link_to "Delete", food, method: :delete, class: "delete-link" %></td>
+        </tr>
```

Comment:
- [ ] Kindly troubleshoot and fix the measurement unit as to why it is not being displayed.


![measure](https://github.com/ikoote1/Recipe-App/assets/53124548/1d8b3cff-d3dd-4b1d-8120-571ee8528122)